src/Microsoft.DotNet.Interactive.Http/Model/HttpNamedRequest.cs

```diff
@@ -233,11 +234,32 @@ internal HttpNamedRequest(HttpRequestNode httpRequestNode, HttpResponse response
     {
         if (currentIndex + 1 == path.Length)
         {
-            var result = responseJSON[path[currentIndex]];
-            return result?.ToString();
+            if(responseJSON.GetType() == typeof(JsonArray))
+            {
+                var jsonArray = (JsonArray)responseJSON;
```

Comment:
```suggestion
                
```

Reply:
This line isn't needed given the previous suggestion, which declares the correctly-typed variable inline.